src/libs/actions/Transaction.js

```diff
@@ -93,6 +93,14 @@ function saveWaypoint(transactionID, index, waypoint) {
             },
         },
     });
+
+    // You can save offline waypoints without verifying the address (we will geocode it on the backend)
+    // We're going to prevent saving those addresses in the recent waypoints though since they could be invalid addresses
+    // However, in the backend once we verify the address, we will save the waypoint in the recent waypoints NVP
+    if (!waypoint.lat || !waypoint.lng) {
```

Comment:
I think this will cause problems if `lat` or `lng` are 0, which are valid values.